src/py-opentimelineio/opentimelineio/schema/__init__.py

```diff
@@ -44,15 +44,11 @@
     image_sequence_reference,
     marker,
     serializable_collection,
-    stack,
     timeline,
-    track,
     transition,
     v2d,
 )
 
-track.TrackKind = TrackKind
```

Comment:
Is this related to this change?

Reply:
Oh, I see. It was sort of odd before to have it in both places. This is cleaner.